packages/test-renderer/src/__tests__/RTTR.methods.test.tsx

```diff
@@ -98,4 +124,14 @@ describe('ReactThreeTestRenderer instance methods', () => {
 
     expect(() => scene.findByProps({ color: 0x0000ff })).toThrow()
   })
+
+  it('should also find three components which are not mounted via r3f and not throw', async () => {
+    const { scene } = await ReactThreeTestRenderer.create(<WithManuallyManagedComponent />)
+    expect(() => scene.findByProps({ name: MANUALLY_MOUNTED_COMPONENT_NAME })).not.toThrow()
```

Comment:
This is important because the test below just tests a _specific_ error but not if no error is thrown at all.